include/xten/Dialect/XTen/XTenOps.td

```diff
@@ -91,6 +91,18 @@ def XTen_SoftmaxOp: XTen_Op<"softmax", []>,
   }];
 }
 
+def XTen_GlobalAveragePool2D: XTen_Op<"globalaveragepool2d", [NoSideEffect]>,
+               Results<(outs AnyTorchTensorType:$output)> {
+  let arguments = (
+    ins AnyTorchTensorType:$input
+  );
+
+  let summary = "GlobalAveragePool2D operator";
+  let description = [{
+    Computes the average pool along the spatial dimensions.
```

Comment:
I think it is important to say that the output has the same rank as the input.

Reply:
done.